formula-test/src/main/java/com/instacart/formula/test/TestFormulaObserver.kt

```diff
@@ -6,7 +6,7 @@ class TestFormulaObserver<Input : Any, Output : Any, FormulaT : IFormula<Input,
     private val delegate: FormulaTestDelegate<Input, Output, FormulaT>,
 ) {
 
-    private var started: Boolean = false
+    @Volatile private var started: Boolean = false
```

Comment:
TODO: given the multi-threaded nature, we will need to go over the internals and ensure that thread memory visibility is configured correctly.